webapp/platform/types/src/properties.ts

```diff
@@ -31,5 +31,6 @@ export type UserPropertyFieldGroupID = 'custom_profile_attributes';
 export type UserPropertyField = PropertyField & {
     type: UserPropertyFieldType;
     group_id: UserPropertyFieldGroupID;
+    attrs?: {sort_order?: number};
 }
-export type UserPropertyFieldPatch = Partial<Pick<UserPropertyField, 'name' | 'attrs' | 'type'>>;
+export type UserPropertyFieldPatch = Partial<Pick<UserPropertyField, 'name' | 'attrs' | 'type' | 'attrs'>>;
```

Comment:
@calebroseland , do we need the second 'attrs'? (I have an upcoming change to this file and stumbled on it). Thanks.

Reply:
@catalintomai Negative, I've removed in a pending branch I'm working on, or feel free to remove in yours.